testing/scripts/deploy_staging.sh

```
@@ -0,0 +1,250 @@
+fi
+
+# ---- Step 6: Fix ownership ----
+
+log "Setting file ownership..."
+ssh "$SSH_HOST" "chown -R www-data:www-data $TARGET"
+pass "Ownership set to www-data"
+
+# ---- Step 7: Run migrations ----
+
+log "Running migrations..."
+MIGRATE_OUTPUT=$(ssh "$SSH_HOST" "docker exec $CONTAINER php $CONTAINER_PATH/migrate.php" 2>&1) || true
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)